docs/velocity/dev/getting-started/pitfalls.mdx

```diff
@@ -33,5 +33,5 @@ public void onProxyInitialization(ProxyInitializeEvent event) {
 
 ## Audience operations not supported
 
-Velocity only supports sending chat messages, action bar messages, titles, and bossbars through the
-Adventure API. No other operations are supported.
+Velocity only supports sending chat messages, action bar messages, titles, bossbars, tablist header and footer, and resource packs through the
```

Comment:
Is it worth having a table here instead?

Reply:
We could, but if Kiory adds something, it won't be listed and would need to be updated.

Reply:
*Literal* or with a component?
I see both in the docs, so I'm not sure what I should use here.